wordle-web/pages/lastTenDays.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <container class="game-container" fluid>
+    <div id="statsContainer"></div>
+  </container>
+</template>
+
+<script setup lang="ts">
+import Axios from "axios"
+  // Function to generate HTML for each day
+  function generateStatsHTML(data) {
+    let html = '';
+    data.forEach(day => {
+      html += `
+        <div class="stats-card">
+          <h3>${day.date}</h3>
+          <p>Total Times Played: ${day.totalTimesPlayed}</p>
+          <p>Total Wins: ${day.totalWins}</p>
+          <p>Total Losses: ${day.totalLosses}</p>
+        </div>
+      `;
+    });
+    return html;
+  }
+
+  // Get container element
+  const statsContainer = document.getElementById('statsContainer');
+
+  // Make API call using Axios
+  const date = '5-20-2024'; // Or use any other date
```

Comment:
```   
const formatDate = format(new Date(), "MM-dd-yyyy");
```

Reply:
Implemented this and it worked very well thank you!